packages/domain-lib/src/commands/index.ts

```diff
@@ -67,6 +68,34 @@ export class ProcessTransferCmd extends CommandMsg {
 	}
 }
 
+// export type CreateSettlementModelCmdPayload = {
+// 	id: string,
```

Comment:
Can we rather keep the `CreateSettlementModelCmdPayload` and then make use of the 
`ISettlementConfig` as the only variable for the `CreateSettlementModelCmdPayload` type?